src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -149,19 +149,46 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
+    const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+    const isCollapsePanelsButton = !panels;
+    /*
+     * When collapsing expandCollapseButton, expandCollapseTeamButton and expandCollapseStudentButton
+     * should be in collapsed state, so that next time when we expand the main panel, we find
+     * the expandCollapseTeamButton and expandCollapseStudentButton in collapsed state.
+     */
+    if (isCollapsePanelsButton && !isButtonInExpandMode) {
+        if ($(expandCollapseStudentsButton).length
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
+            replaceButtonHtmlAndTooltipText(expandCollapseStudentsButton, STRING_EXPAND, STRING_COLLAPSE);
+        }
+        if ($(expandCollapseTeamButton).length
+            && $(expandCollapseTeamButton).html().trim().startsWith(STRING_EXPAND)) {
+            replaceButtonHtmlAndTooltipText(expandCollapseTeamButton, STRING_EXPAND, STRING_COLLAPSE);
+        }
+    }
+    // When collapsing expandCollapseTeamButton, expandCollapseStudentButton should be in collapsed state.
+    if ($(expandCollapseStudentsButton).length && (expandCollapseButton === $(expandCollapseTeamButton))
+        && $(expandCollapseTeamButton).html().trim().startsWith(STRING_COLLAPSE)
```

Comment:
This condition can be removed if we move the whole if block into the else block of `if (isButtonInExpandMode) {` below.